.github/workflows/release.yml

```diff
@@ -1,16 +1,18 @@
 name: release
-
 on:
   push:
     tags:
       - 'v*'
-  pull_request:
+  pull_request_target:
```

Comment:
This one is the real dangerous method 🤣 

It will pass the write permission to fork repos.

Reply:
Sorry, this is a test of several ways to solve that problem, after nearly a hour, I concluded that:
Leaving it as it is is probably the best thing to do.